.github/workflows/cache_data.yaml

```
@@ -3,7 +3,7 @@ name: Cache data
 
 on:
   # Uncomment the 'pull_request' line below to manually re-cache data artifacts
-  # pull_request:
+  pull_request:
```

Need to comment this line before merging:
```suggestion
  # pull_request:
```